src/redux/reducers/employer.ts

```diff
@@ -24,6 +27,7 @@ const initialState = {
     seniority: '',
     skills: [],
   },
+  jobPostIdToDelete: '',
```

Comment:
I think we don't need to store the id inside reducer. There's a way that we can receive this `id` from action without storing it inside reducer